### PR TITLE
Support itemTake and itemDrop

### PR DESCRIPTION
**Pull request by jmarianer**

### CODE REVIEW

File: test_all.py [hunk 1]
```diff
@@ -893,10 +893,10 @@ def test_unit_actions():
     ucontrol targetp @unit 1 0 0 0
     ucontrol target 9 10 1 0 0
     ucontrol target 0 0 0 0 0
-    ucontrol itemTake from item amount 0 0
-    ucontrol itemTake from item amount 0 0
-    ucontrol itemDrop to amount 0 0 0
-    ucontrol itemDrop to amount 0 0 0
+    ucontrol itemTake "container1" "copper" 1 0 0
```
Comment (Lonami) [inline test_all.py:896]: Does this work in-game? Or would it need to be

```
ucontrol itemTake container1 @copper 1 0
```

?

**Review by Lonami — COMMENTED** [review]
Thanks! I have a question before we can merge this.

File: test_all.py (hunk 1 above)
Comment (jmarianer) [reply test_all.py]: You're absolutely right. I was asleep at the wheel when fixing that test. It's fixed now.

**Comment by Lonami** [comment]
Do you know if `"container1"` (as a string) works or would it need to be `container1` (a reference)?

**Comment by jmarianer** [comment]
Thanks for catching this. I swear I did that but I must have done a `commit --amend` and forgotten to force-push.

**Pull request merged**